src/uniprotkb/components/entry/tabs/genomic-coordinates/GenomicLoc.tsx

```diff
@@ -0,0 +1,48 @@
+import { LongNumber } from 'franklin-sites';
+
+import ExternalLink from '../../../../../shared/components/ExternalLink';
+
+import { GenomicLocation } from './types';
+
+export const getEnsemblLink = (
+  taxID: number,
+  chromosome?: string,
+  start?: number,
+  end?: number
```

Comment:
These are optional but if they were undefined the url wouldn't make sense - do we need to check them before url formation?